modules/apps/wysiwyg/wysiwyg-web/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+Bundle-Name: Liferay Wysiwyg Web Conversion
+Bundle-SymbolicName: com.liferay.wysiwyg.web
+Bundle-Version: 1.0.0
+Liferay-Releng-Module-Group-Description:
+Liferay-Releng-Module-Group-Title:
+Web-ContextPath: /wysiwyg-web
```

Comment:
I'm not sure if a -web module is the most appropriate. Can you check with Shuyang?